src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,57 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class WorkWithFile {
+    static final String SEPARATOR = ",";
+    static final int DESCRIPTION_OF_THE_LIST = 0;
+    static final int VALUE_OF_THE_LIST = 1;
+    static final String SUPPLY = "supply,";
+    static final String BUY = "buy,";
+    static final String RESULT = "result,";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(makeReport(readFromFile(fromFileName)), toFileName);
+    }
+
+    private List<String> readFromFile(String fromFileName) {
+        String content = null;
+        try {
+            content = Files.readString(Paths.get(fromFileName));
```

Comment:
```suggestion
            return Files.readAllLines(Paths.get(fromFileName));
```

Reply:
so I need the readFromFile method to return a String? that I will convert to a List in the next method(makeReport)?

Reply:
or like **return new ArrayList<String>(Arrays.asList(Files.readString(Paths.get(fromFileName)).split("\n")));**